src/main.cpp

```diff
@@ -351,6 +353,8 @@ int main(int argc, char **argv) {
             world.updateGameObjects(deltaTime, totalTime);
             camera.update(deltaTime);
 
+            gameManager.decreaseTime(deltaTime);

```

Comment:
We'll also need some way to end the game if the player reaches the 'end' of a level.  Probably a box the player collides with that triggers end of game. Not needed for this PR, however.